dev/teamcity/dist-publish-assets-tc

```diff
@@ -49,13 +44,8 @@ sed -e "s|<%build_number%>|$BUILD_NUMBER|" \
     -e "s|<%branch%>|$BUILD_VCS_BRANCH|" \
     static/build-template.json | tee $static_folder/build.json
 
+aws s3 cp --acl bucket-owner-full-control --region=eu-west-1 --recursive $static_folder s3://$RIFF_RAFF_ARTIFACT_BUCKET/dotcom:static/$BUILD_NUMBER
 aws s3api put-object --acl bucket-owner-full-control --region=eu-west-1 --bucket $RIFF_RAFF_BUILD_BUCKET --key dotcom:static/$BUILD_NUMBER/build.json  --body $static_folder/build.json
-aws s3api put-object --acl bucket-owner-full-control --region=eu-west-1 --bucket $RIFF_RAFF_ARTIFACT_BUCKET --key dotcom:static/$BUILD_NUMBER/artifacts.zip --body $static_folder/artifacts.zip
-
-set +o xtrace
-echo "##teamcity[publishArtifacts '$static_folder/artifacts.zip => static']"
-echo "##teamcity[publishArtifacts '$static_folder/build.json => static']"
-set -o xtrace
```

Comment:
I've done the same for all of the other artifacts (albeit through a change in behaviour in `sbt-riffraff-artifact` in 0.8.4 - https://github.com/guardian/sbt-riffraff-artifact/pull/20). I sent an e-mail earlier asking whether people wanted them - perhaps that wasn't the best channel for the question though.